modules/acs-architecture-connections.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * architecture/acs-architecture.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-architecture-connections_{context}"]
+= {product-title-short} connection protocols and default ports
+
+The protocols and default ports for connections within {product-title-short} and between {product-title-short} and external components are shown in the following table.
+
+.{product-title-short} connections between components
+
+[cols="3",options="header"]
+|===
+| Component or external entity | Connection type | Port
+
+| Central and external services, for example, Jira and PagerDuty
+| HTTPS
+| ?
+
+| Central and secured cluster
+| HTTPS
+| 443
+
+| Central and sensor
+| TCP
+| 443
```

Comment:
are these different?

Reply:
Actually, I guess TCP is the physical and HTTP is the protocol on top, and they don't need to be broken out?

Reply:
Inside the Secured Cluster, only two components reach out to Central:

* Sensor - has a persistent gRPC connection to Central. Port is 443 (I think)
* Admission Controller - rare, but it is sometimes allowed to reach out to Central, directly. Probably HTTP 443, but I actually don't know

Reply:
We should probably differentiate between HTTP(S) and gRPC. 
The latter can be filtered out by Layer 7 Firewalls / WAFs, so it would need to be explicitly allowed in such environments.
Primarily, Sensor and Central communicate over a bidirectional gRPC stream, initiated by Sensor to Central's port 443.